src/windows/CameraProxy.js

```diff
@@ -165,7 +165,16 @@ module.exports = {
 
                             var storageFolder = Windows.Storage.ApplicationData.current.localFolder;
                             file.copyAsync(storageFolder, file.name, Windows.Storage.NameCollisionOption.replaceExisting).then(function (storageFile) {
-                                successCallback(URL.createObjectURL(storageFile));
+                                var res;
+                                if (destinationType == Camera.DestinationType.NATIVE_URI) {
+                                    res = storageFile.path
+                                        .replace(/\\/g, '/')
+                                        .replace(':','%3A');
```

Comment:
Colon is invalid character in URI. This replacement makes returned URI compatible with native URI suitable for File plugin.